content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -27,13 +27,10 @@ Before you begin, you'll need to prepare data files and a Linux instance.
 ### Data Files
 
 * TLS private key and certificate
-  * The installer allows for using a self-signed certificate but HashiCorp does
-    _not_ recommended this. Your VCS provider will likely reject that certificate
-    when sending webhooks. If you do use the self-signed certificate, you must configure
-    each webhook to ignore SSL errors within your VCS provider.
+  * The installer allows for using a publicly trusted certificate. If you do not use a publicly trusted certificate, your VCS provider will likely reject that certificate when sending webhooks. 
```

Comment:
"allows for" should probably change to something else, since this isn't really a caveat anymore. Maybe something like "A publicly-trusted certificate is recommended. If you do not..."

Reply:
👍